tests/keras_contrib/backend/backend_test.py

```diff
@@ -151,14 +153,21 @@ def test_moments(self):
                     ip_tf = KTF.variable(ip)
                     tf_mean, tf_var = KCTF.moments(ip_tf, axes, keep_dims=keep_dims)
 
+                    # ip_cntk = KCTK.variable(ip)
+                    # cntk_mean, cntk_var = KCNTK.moments(ip_cntk, axes, keep_dims=keep_dims)
+
                     th_mean_val = KTH.eval(th_mean)
                     tf_mean_val = KTF.eval(tf_mean)
+                    # cntk_mean_val = KCTK.eval(cntk_mean)
                     th_var_val = KTH.eval(th_var)
                     tf_var_val = KTF.eval(tf_var)
+                    # cntk_var_val = KCTK.eval(cntk_var)
 
                     # absolute tolerance needed when working with zeros
                     assert_allclose(th_mean_val, tf_mean_val, rtol=1e-4, atol=1e-10)
                     assert_allclose(th_var_val, tf_var_val, rtol=1e-4, atol=1e-10)
+                    # assert_allclose(th_mean_val, cntk_mean_val, rtol=1e-4, atol=1e-10)
```

Comment:
should probably correct the commented lines in this file

Reply:
The issue is that CNTK tests won't pass for some reason. It is using Contrib's `K.moments()` call, and technically, it should do just fine on CNTK, but it fails. And now that Contribs tests are broken, we cant check properly either.